README.md

```diff
@@ -5,82 +5,97 @@ grandchildrice benchmarks focuses on ZK-VMs capable of executing Rust programs (
 
 The goal of the current benchmarks is to compare ZK-VMs in a context of client-side proving on consumer devices: laptop and mobile. The on-chain verification is discarded and proof size is less important.
 
-As the language used to define programs might not exactly have the same features, an *applicative* approach has been taken: the benchmarked program is the most optimized version for a given zkDSL/ZK-VM.
+As the language used to define programs might not exactly have the same features, an _applicative_ approach has been taken: the benchmarked program is the most optimized version for a given zkDSL/ZK-VM.
 For example, the fibonacci program will use the native field of the ZK-VM when doable.
 
 ## Benchmark state
-|ZK-VM|Fibonacci|
-|-|-|
-|Cairo M | ✅ |
-|Miden |✅|
-|Noir ProveKit|✅|
-|RiscZero|✅|
-|SP1|✅|
-|ZKM|✅|
 
-The following ZK-VMs are yet to be adapted: Cairo 252, Ceno, Jolt, Nexus, Noir Barretenberg, OpenVM, Pico, Valida
+| ZK-VM         | Fibonacci |
+| ------------- | --------- |
+| Cairo M       | ✅        |
+| Miden         | ✅        |
+| Noir ProveKit | ✅        |
+| OpenVM        | ✅        |
+| RiscZero      | ✅        |
+| SP1           | ✅        |
+| ZKM           | ✅        |
+
+The following ZK-VMs are yet to be adapted: Cairo 252, Ceno, Jolt, Nexus, Noir Barretenberg, Pico, Valida
```

Comment:
I think we can remove Noir Barretenberg as provekit is already there with better perf

Reply:
indeed good point